src/transformStateWithClones.js

```diff
@@ -8,6 +8,36 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const clonedState = [];
+  const currentStateData = { ...state };
+  const addDataToArray = () => {
+    clonedState.push({ ...currentStateData });
+  };
```

Comment:
jesus, is that chatgpt helped you achieve this? 😄

Reply:
No, just watching too many bad youtube channels I think)
I can't use the arrow function or what's wrong here?